tests/autoyast/installation.pm

```diff
@@ -343,7 +343,8 @@ sub run {
             return;
         }
         elsif (match_has_tag('opensuse-welcome')) {
-            return;
+            assert_and_click 'dont-show-on-next-startup';
+            assert_and_click 'close-welcome';
```

Comment:
This should be in a function to be reused.

For example, here: https://openqa.opensuse.org/tests/1077390#step/multi_users_dm/16